common/client-core/src/error.rs

```diff
@@ -36,6 +36,9 @@ pub enum ClientCoreError {
     #[error("no gateway with id: {0}")]
     NoGatewayWithId(String),
 
+    #[error("Invalid URL: {0}")]
+    InvalidURL(String),
```

Comment:
Sorry for being picky about it, but that capitalization makes my right eye twitch ;)
Convention in Rust is InvalidUrl

Reply:
I agree. However, this was taken as is from the GatewayClient - https://github.com/nymtech/nym/blob/9d014742776d880ad5f467868fed226428a80c4c/common/client-libs/gateway-client/src/error.rs#L45-L48

So there is precedent in this case! I can change both though if we want lol.